devel/cpp-argparse-dev/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+github.setup            rue-ryuzaki argparse 1.6.5 v
+github.tarball_from     archive
+revision                0
+name                    cpp-argparse-dev
+categories              devel
+platforms               darwin
+supported_archs         noarch
+license                 MIT
+maintainers             {gmail.com:golubchikov.mihail @rue-ryuzaki}
+description             C++ argument parser.
+long_description        Python-like header-only argument parser for C++ projects.
+
+homepage                https://github.com/rue-ryuzaki/argparse
```

Comment:
```suggestion
```

This is set by the GitHub PortGroup, and so isn't required.

https://github.com/macports/macports-ports/blob/ba05278954176baf5844ef6b5d290b3cdfe3516f/_resources/port1.0/group/github-1.0.tcl#L14